experimental/ExtendTeamsforM365/client/pages/categories.html

```diff
@@ -15,7 +24,7 @@
     <top-nav-panel></top-nav-panel>
 
     <h3>Northwind Product Categories</h3>
-    <table id="categories">
+    <table id="categories" style="overflow: auto;">
```

Comment:
Move inline css to css file.

Reply:
Changed and shifted to northwind-teams.css